lib/sas-master.js

```diff
@@ -74,15 +78,27 @@ async function getYnabTransactions(ynabApi, budgetId, accountId, earliestDate) {
   return transactions.data.transactions
 }
 
+function isTransactionNotInStatement(ynabTransaction, transactions) {
+  return !isTransactionInStatement(ynabTransaction, transactions)
+}
```

Comment:
This is an unnecessary function, since it just negates `isTransactionInStatement` =/